projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -515,6 +517,15 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
 
   private onMouseMove(): void {
     const locationData = this.getMouseDataForCurrentEvent();
+
+    if (this.sync && this.groupId) {
```

Comment:
We're tying two things together in the implementation in an unclear way - not only do these flags need to be set, we're assuming that setting them means we should fire on hover, and if these aren't set, the hover listeners get ignored. My suspicion is that at this level, we'd want a distinct listener rather than any of these three things. The group id can be built into it, and its presence indicates it should fire.